config/jobs/kubernetes/sig-k8s-infra/trusted/releng/releng-trusted.yaml

```diff
@@ -47,6 +47,13 @@ postsubmits:
         - cip
         - --thin-manifest-dir=/home/prow/go/src/github.com/kubernetes/k8s.io/k8s.gcr.io
         - --confirm
+      resources:
+        requests:
+          cpu: 2
+          memory: "4Gi"
```

Comment:
why not add a bit more? or we want to check how looks like if this and if need to bump again?

Reply:
The latter.I want to monitor consumption for the next PR merges.